email_mailbox/src/utils/electronInterface.js

```diff
@@ -52,3 +62,18 @@ export const cleanDataLogout = async recipientId => {
   await cleanData(recipientId);
   return createSignalTables();
 };
+
+/*  Firebase
+----------------------------- */
+ipcRenderer.on(TOKEN_UPDATED, async (_, token) => {
```

Comment:
use better-ipc

Reply:
Firebase management library use elecron's ipc by default because it uses Window instance for setup. 
See:
```
electron_app/src/windows/mailbox.js
```
(setupPushReceiver)